sender_test.go

```diff
@@ -63,7 +68,7 @@ func TestSenderRetries(t *testing.T) {
 		recoverCalls = nil
 		sender = &testAmqpSender{
 			sendErrors: []error{
-				amqp.ErrLinkClosed,
```

Comment:
Do we need a test to ensure that we don't recover on a closed link, or is that already covered?

Reply:
Done!